src/cljfx/context.clj

```diff
@@ -112,21 +124,24 @@ Possible reasons:
     ::context
     (assoc deps k v)))
 
+(defn- register-cache-entry [context *cache cache sub-id]
+  (if (has? cache sub-id)
+    (swap! *cache hit sub-id)
+    (let [recalc #(swap! *cache miss sub-id (calc-cache-entry context sub-id))]
+      (if (has? cache [::dirty sub-id])
+        (if (*processing-dirty* sub-id)
+          (do (swap! *cache evict [::dirty sub-id])
+              (recalc))
+          (binding [*processing-dirty* (conj *processing-dirty* sub-id)]
+            (sub-from-dirty context *cache cache sub-id)))
```

Comment:
Are you sure we need `*processing-dirty*`? I tried replacing this `(if (*processing-dirty sub-id) ...)` with just `(sub-from-dirty ...)` and it seems to work. Can you make a test case where it's needed?